packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks.ts

```diff
@@ -58,14 +59,12 @@ const stockLineToDraftLine = (
 ): DraftStocktakeLine => {
   return {
     stocktakeId,
-    itemCode: '',
-    itemName: '',
     countThisLine: false,
     isCreated: false,
     isUpdated: false,
     ...line,
     snapshotNumberOfPacks: line.totalNumberOfPacks,
-    expiryDate: line.expiryDate ? new Date(line.expiryDate) : null,
+    expiryDate: line.expiryDate ? line.expiryDate : null,
```

Comment:
I removed the parsing of dates in the query into date objects to avoid the performance problem we were seeing with invoices and non-serialisable data